src/pybind/mgr/orchestrator.py

```diff
@@ -30,6 +30,16 @@ def result(self):
         """
         raise NotImplementedError()
 
+    @property
+    def exception(self):
+        # type: () -> Exception
+        """
+        Holds an exception object. Used for deployment, configuration or user error.
+
+        TODO: does this duplicate is_errored?
```

Comment:
I'd say it does. Maybe get rid of is_errored, or turn those calls into a check to see whether the exception property is None?

Reply:
like so?